src/Compiler/Service/ServiceParsedInputOps.fs

```diff
@@ -1312,7 +1330,7 @@ module ParsedInput =
             TryGetCompletionContextInPattern suppressIdentifierCompletions pat1 None pos
             |> Option.orElseWith (fun () -> TryGetCompletionContextInPattern suppressIdentifierCompletions pat2 None pos)
         | SynPat.IsInst (_, m) when rangeContainsPos m pos -> Some CompletionContext.Type
-        | SynPat.Wild m when rangeContainsPos m pos -> Some CompletionContext.Invalid
+        | SynPat.Wild m when rangeContainsPos m pos && m.StartColumn <> m.EndColumn -> Some CompletionContext.Invalid
```

Comment:
Just curious, why is this needed? When would this come into play?

Reply:
Hmm good to know, thanks!

Reply:
> They're not even wrapped in SynPat.FromParserError.

We should probably either wrap them or add a union case that represents an error at a range without including another pattern inside of it, similar to `SynExpr.ArbitraryAfterError`.

Reply:
@nojaf, do you have an opinion on this?

Reply:
Side note, nojaf is on vacation, it might take them long time to respond. :)

Reply:
Alright, so do we merge this and return to revamp the represantation of errors in patterns later?

Reply:
Yes, I don't mind merging it this way.

Reply:
The top 2 completions are [suggested names](https://github.com/dotnet/fsharp/pull/15530).

Reply:
Oh I see.